src/main/java/uk/gov/hmcts/reform/civil/utils/DateUtils.java

```diff
@@ -14,6 +14,11 @@ private DateUtils() {
         //No op
     }
 
+    public static String formatDate(LocalDateTime date) {
```

Comment:
Duplicated method, exact the same as the next one

Reply:
one is with LocalDate and the other is wit LocalDateTime